src/Components/LoginPage/LoginPage.js

```diff
@@ -21,20 +63,26 @@ const LoginPage = () => {
           type='text'
           placeholder='Username'
           name='username'
-          // value={username}
-          // onChange={handleChange}
+          value={signInData.username}
+          onChange={handleChange}
           className="input"
         />
         <input
           type='password'
           placeholder='Password'
           name='password'
-          // value={password}
-          // onChange={handleChange}
+          value={signInData.password}
+          onChange={handleChange}
           className="input"
         />
         <button onClick={event => handleSubmit(event)}>Login</button>
```

Comment:
What happens if there's nothing typed or a space typed and you click login?

Reply:
If nothing is typed or space is typed, and the user clicks login, the user will see a message that says that the username/password was incorrect and the inputs are reset. The onChange updates the value as the space or an empty string if the user does not put anything in the input field. The handleSubmit checks to see if the values are valid usernames and passwords.